src/components/DeviceList/index.js

```diff
@@ -1,18 +1,20 @@
 import React from 'react'
 import P from 'prop-types'
 import DeviceItem from '../DeviceItem'
+import { DevListUl } from './style.js'
 import * as S from '../DeviceItem/style.js'
 
 const DeviceList = props => {
   const { devices, onDelete, onChangeActive, onSave } = props
   return (
     <div className="deviceList">
-      <ul>
+      <DevListUl>

```

Comment:
Please don't use the html element's name as a suffix to a styled component. DevList is a good name